lib/src/app/cast.js

```diff
@@ -18,58 +20,73 @@ import '../styles/player.css';
  * @param {PageLevelParameters} [pageLevelParameters]
  */
 export function init(namespace, pageLevelParameters) {
-    var mediaManager = null;
-    var castPlayer = new CastPlayer(namespace, pageLevelParameters);
-
     logger.setLevel(process.env.LOG_LEVEL);
     UIManager.setStatusCast(UIManager.status.LOADING);
 
-    // create a new instance of the Media Manager to interact with the sender app request
-    mediaManager = new cast.receiver.MediaManager(castPlayer);
-
-    // backup of the original onLoad event handler to use it after our custom logic
-    mediaManager.origOnLoad = mediaManager.onLoad;
+    const castPlayer = new CastPlayer(namespace, pageLevelParameters);
+    const mediaManager = new cast.receiver.MediaManager(castPlayer);
 
     /**
-     * onLoad assign a new implementation of the onLoad event that can sets the needed information from the
+     * Exotic class method extending by Google devs
+     * @see https://developers.google.com/cast/docs/custom_receiver (section Media)
+     * onLoad assign a new implementation of the onLoad event that can sets the
+     * needed information from the
```

Comment:
Change this to:
Exotic class method extending by Google devs
     * @see https://developers.google.com/cast/docs/custom_receiver (section Media)
     * onLoad assigns a new implementation of the onLoad event that can set the
     * needed information from the incoming event

Reply:
+